lib/notifications/fcm/fcm_notifications_listener.dart

```diff
@@ -231,10 +197,44 @@ class FcmNotificationsListener {
       }
     }
   }
+
+  /// Handle the click event on notification when app is in background and terminated.
+  Future<void> _onBackgroundMessageOpenedHandler(RemoteMessage? message) async {
+    if (message != null) {
+      print('FCM - Terminated app, notification tap handler with message: ' +
```

Comment:
Can we call the NotificationConsumer here? Especially on Android where code runs in different isolate?